definitions/npm/reselect_v3.x.x/flow_v0.104.x-/test_reselect_3.x.x.js

```diff
@@ -241,8 +241,7 @@ createSelector(
     return x + y;
   }
 ).resultFunc(
-  { x: 100, y: 200 },
-  {x: 10},
-  10,
+  100,
+  200,
```

Comment:
I don't understand reselect that well, can you explain why the test needed to be changed as well?

Reply:
resultFunc is called with the results of the input selectors not the state and props so the test has been updated to reflect this.